frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx

```diff
@@ -108,7 +108,7 @@ export default function SaveImage({
       }
       link.click();
       link.remove();
-      track(EVENTS.WMG_DOWNLOAD_COMPLETE, { file_type: fileType });
+      track(EVENTS.WMG_DOWNLOAD_COMPLETE, { file_type: fileType , tissues: selectedTissues.toString(), genes: selectedGenes.toString() });
```

Comment:
Just double checking, did Amanda say if she wants us to send the tissue count and gene count for her as well?

Thank you!

Reply:
No she said this was all she needed :)

Reply:
Awesome! 

Last question: did she say anything about the order of the array? E.g., does she care about [A, B] vs. `[B, A]`? Just wondering if it'd be easier for her to identify the common tissues and genes combo if we sorted the arrays for her, or if she wants to preserve the user selected order or something 😄

Reply:
Oh 🤔 I am under the assumption that they come in ordered by top to bottom, is that wrong? Do they come in by order they clicked?

Reply:
Ah yeah just tested and the order is by clicked!

Reply:
Confirmed by clicked is good! Thanks for bringing that up!

Reply:
awesomee thanks so much for confirming, @dsadgat 🤩 🚀 !